aws/terminator/security_services.py

```diff
@@ -265,6 +265,27 @@ def terminate(self):
         self.client.delete_regex_pattern_set(RegexPatternSetId=self.id, ChangeToken=self.change_token)
 
 
+class WafV2IpSetRegionl(Terminator):
+    @staticmethod
+    def create(credentials):
+        return Terminator._create(credentials, WafV2IpSetRegionl, 'wafv2', lambda client: client.list_ip_sets(Scope='REGIONAL')['IPSets'])
```

Comment:
This isn't working with multiple creates in one class, there's no way to differentiate the scopes in the output when testing/troubleshooting (less critical but it would be nice to have) and the method is not able to be executed properly when resources exist.  I tried tweaking it a bit but wasn't able to get it working.  I know it's a lot of boilerplate and copy/pasta but is there a reason not to make 6 classes?

Tested while running the wafv2 test suite in 450:
```
(.terminator) jill@lab:~/src/aws-terminator/aws$ python3 cleanup.py --target WafV2IpSet --target WafV2RuleGroup --target WafV2WebAcl --stage dev -vvv -c
cleanup     : DEBUG    located WafV2IpSet: count=0
cleanup     : DEBUG    located WafV2IpSet: count=0
cleanup     : DEBUG    located WafV2RuleGroup: count=1
cleanup     : ERROR    {"message": "exception processing resource type: <class 'terminator.security_services.WafV2RuleGroup'>", "traceback": "Traceback (most recent call last):\n  File \"/home/jill/src/aws-terminator/aws/terminator/__init__.py\", line 86, in cleanup_test_account\n    instances = terminator_type.create(credentials)\n  File \"/home/jill/src/aws-terminator/aws/terminator/security_services.py\", line 312, in create\n    item.update({\"Scope\": \"REGIONAL\"})\nAttributeError: 'WafV2RuleGroup' object has no attribute 'update'"}
cleanup     : DEBUG    located WafV2WebAcl: count=1
cleanup     : ERROR    {"message": "exception processing resource type: <class 'terminator.security_services.WafV2WebAcl'>", "traceback": "Traceback (most recent call last):\n  File \"/home/jill/src/aws-terminator/aws/terminator/__init__.py\", line 86, in cleanup_test_account\n    instances = terminator_type.create(credentials)\n  File \"/home/jill/src/aws-terminator/aws/terminator/security_services.py\", line 328, in create\n    item.update({\"Scope\": \"REGIONAL\"})\nAttributeError: 'WafV2WebAcl' object has no attribute 'update'"}
```